### PR TITLE
feature: Add installation docs

### PR DESCRIPTION
**Pull request by mrfyda**

### CODE REVIEW

File: configuration/monitoring.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/podmonitor.crd.yaml
+​
+sleep 5 # wait for crds to be created
+​
+helm upgrade --install monitoring stable/prometheus-operator \
+  --version 6.9.3 \
+  --namespace monitoring \
+  --set prometheusOperator.createCustomResource=false \
+  --set grafana.service.type=LoadBalancer \
+  --set grafana.adminPassword="SOMETHING_TO_BE_CHANGED"
```
Comment (joaocosta-codacy) [inline configuration/monitoring.md]: For consistency with the ones presented above, maybe this adminPassword should say "PLEASE_CHANGE_ME"?
I am just thinking that, myself, as a developer, if I skim through the text and I see some CHANGE_ME in some credentials, I will try to do a find to understand what else needs to be replaced.

File: installation/cloud/eks.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+### Scripted cluster creation
+
+A [bootstrap script](https://gitlab.com/gitlab-org/charts/gitlab/blob/master/scripts/eks_bootstrap_script)
+has been created to automate much of the setup process for users on EKS.
+
+The script will:
+
+1. Create a new EKS cluster.
+1. Setup `kubectl`, and connect it to the cluster.
```
Comment (joaocosta-codacy) [inline installation/cloud/eks.md]: 1. 
2.
3.

instead of 
1.
1.
1.

**Comment by mrfyda** [comment]
Thanks for the review @joaocosta-codacy!

File: installation/cloud/eks.md (hunk 2 above)
Comment (mrfyda) [reply installation/cloud/eks.md]: Same as above

File: installation/cloud/eks.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+1. Initialize Helm and install Tiller.
+
+The script uses [eksctl](https://eksctl.io) to initialize the cluster. If it cannot locate it in your PATH, it will install it
```
[the remark this answers is not in the document]
Comment (mrfyda) [reply installation/cloud/eks.md]: done

**Comment by paulopontesm** [comment]
This should be inside "docs/"

**Pull request merged**